install.sh

```diff
@@ -7,8 +7,14 @@ cd ../../
 composer require guzzlehttp/guzzle:^6.0
 composer dump-autoload
 
+sys_dir=$(awk -F "=" '/^sys_dir/ {gsub(/[ \t]/, "", $2); print $2}' /etc/lms/lms.ini)
+
+mkdir cron
+echo "0,5,10,15,20,25,30,35,40,45,50,55 * * * * bash -lc ${sys_dir}/plugins/BillTech/bin/billtech-update-links.php | ${sys_dir}/plugins/BillTech/bin/timestamp.sh >> /var/log/billtech/\`date +\%Y\%m\%d\`-update-links.log 2>&1" >> cron/billtech-update-links-cron
+echo "* * * * * bash -lc ${sys_dir}/plugins/BillTech/bin/billtech-update-payments.php | ${sys_dir}/plugins/BillTech/bin/timestamp.sh >> /var/log/billtech/\`date +\%Y\%m\%d\`-update-payments.log 2>&1" >> cron/billtech-update-payments-cron
```

Comment:
Na końcu pliku jest linia która robiła to samo przed zmianą. Co było nie tak z tym?

Reply:
Te 2 linijki mają na celu dynamiczne stworzenie 2 plików cron z uwzględnieniem poprawnej ścieżki do projektu, zamiast `/var/www/html` na sztywno. Nie wiedziałem jak inaczej zaimportować do tych plików stałą z lms.ini.

Reply:
Ok, rozumiem. A jakby to zrobić w tych plikach tak jak było to by nie zadziałało z tym ${sys_dir}?